bedrock/mozorg/templates/mozorg/about/leadership.html

```diff
@@ -66,8 +66,8 @@ <h2 class="section-title">{{ _('Management Teams') }}</h2>
 
     <h3 class="group-title" id="mgmt-corp"><a href="{{ url('foundation.moco') }}">{{ _('Mozilla Corporation') }}</a></h3>
 
-    <div class="gallery">
-      <article id="mitchell-baker" class="vcard has-bio" itemscope itemtype="http://schema.org/Person">
+    <div class="gallery mgmt-corp">
+      <article id="mitchell-baker" data-id="mitchell-baker" class="vcard has-bio" itemscope itemtype="http://schema.org/Person">
```

Comment:
Note for reviewer: we can't rely on using the ID for testing what's displayed in the modal as the [page JS removes it](https://github.com/mozilla/bedrock/blob/master/media/js/mozorg/about-leadership.js#L41) to prevent duplicate ID's on the page, so instead I'm adding a data attribute to use for reference.